tools/ardusub.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/python
+
+from pymavlink import mavutil
+import time
+
+master = mavutil.mavlink_connection('udpout:localhost:9000')
+
```

Comment:
This script should timeout after (~3s) and exit with a return code indicating if the version was successfully communicated. Make 3 attempts, 1 second sleep.